mail/rspamd/Portfile

```diff
@@ -74,24 +73,24 @@ cmake.set_osx_architectures no
 # Reference: https://github.com/rspamd/rspamd/blob/master/CMakeLists.txt
 # Also see https://github.com/rspamd/rspamd/issues/2884
 configure.args-append \
-    -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
-    -DCMAKE_INSTALL_PREFIX=${prefix} \
-    -DCONFDIR=${prefix}/etc/${name} \
-    -DDBDIR=${prefix}/var/lib/${name} \
+    -DENABLE_LUAJIT=ON \
+    -DENABLE_TORCH=ON \
+    -DENABLE_SNOWBALL=ON \
     -DENABLE_FANN=ON \
     -DENABLE_GD=ON \
     -DENABLE_LIBUNWIND=ON \
-    -DENABLE_LUAJIT=ON \
-    -DENABLE_SNOWBALL=ON \
-    -DENABLE_TORCH=ON \
-    -DINSTALL_EXAMPLES=ON \
-    -DLIBDIR=${prefix}/lib \
-    -DLOGDIR=${prefix}/var/log/${name} \
-    -DMANDIR=${prefix}/share/man \
+    -DENABLE_HYPERSCAN=ON \
     -DNO_SHARED=ON \
-    -DPCRE_ROOT_DIR=/usr/lib \
+    -DINSTALL_EXAMPLES=ON \
+    -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
     -DRSPAMD_USER=${rspamd_user} \
-    -DRUNDIR=${prefix}/var/run/${name}
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCONFDIR=${prefix}/etc/${name} \
+    -DMANDIR=${prefix}/share/man \
+    -DRUNDIR=${prefix}/var/run/${name} \
+    -DDBDIR=${prefix}/var/lib/${name} \
+    -DLOGDIR=${prefix}/var/log/${name} \
+    -DLIBDIR=${prefix}/lib
```

Comment:
Are you unsorting these for a reason?

Reply:
Yes, to parallel the dependencies and the configure options to try to keep track of stuff.